test/lightning/usage_tracking/worker_test.exs

```diff
@@ -58,7 +58,7 @@ defmodule Lightning.UsageTracking.WorkerTest do
       report = Report |> Repo.one()
 
       assert %Report{submitted: true} = report
-      assert DateTime.diff(DateTime.utc_now(), report.submitted_at, :second) < 1
+      assert DateTime.diff(DateTime.utc_now(), report.submitted_at, :second) <= 1
```

Comment:
Good spotting on the time sensitive test failure, I think this can be shortened to `DateTime.utc_now() > report.submitted_at`, afaik that works on the maximum available precision.